docs/sources/panels/visualizations/graph-panel.md

```diff
@@ -0,0 +1,155 @@
++++
+title = "Graph panel"
+keywords = ["grafana", "graph panel", "documentation", "guide", "graph"]
+type = "docs"
+aliases = ["/docs/grafana/latest/reference/graph/", "/docs/grafana/latest/features/panels/graph/"]
+[menu.docs]
+parent = "visualizations"
+weight = 100
+draft = "true"
++++
+
+# Graph panel
+
+This visualization is the most-used in the Grafana ecosystem. It can render as a line, a path of dots, or a series of bars. This type of graph is versatile enough to display almost any time-series data.
+
+## Data and field options
+
+Graph visualizations allow you to apply:
+- Data transformations
+- Alerts - This is the only type of visualization that allows you to set alerts. For more information, see [LINK TO UPDATED ALERTING TOPIC]
+- [Thresholds]({{< relref "../thresholds.md" >}})
+
+## Display options
+
+Use these settings to refine your visualization.
+
+- **Bars -** Display values as a bar chart.
+- **Lines -** Display values as a line graph.
+- **Line width -** The width of the line for a series. (default 1).
+- **Staircase -** Draws adjacent points as staircase.
+- **Area fill -** Amount of color fill for a series. (default 1, 0 is none)
+- **Fill gradient -** XXX
+- **Points -** Display points for values.
+- **Point radius -** Controls how large the points are.
+
+### Hover tooltip
+XXX - this entire section
+- **Mode**
+  - **All series -**
+  - **Single -**
+- **Sort order**
+  - **None -**
+  - **Increasing -**
+  - **Decreasing -**
+
+### Stacking and null value
+
+- **Stack -** Each series is stacked on top of another.
+- **Percent -** Available when **Stack** is selected. Each series is drawn as a percentage of the total of all series.
+- **Null value -** How null values are displayed.
```

Comment:
This is a very important setting. Not sure if this is the place to signal that. The default is set to null and there is a reason for that, we don't want to mislead anyone.

If monitoring a server's CPU load and the load reaches 100% then the server will lock up and the agent sending statistics will not be able to collect the load statistic. This will lead to a gap in the metrics and having the default as null will show the gaps and indicate that something is wrong. If the default was connected, then it would be easy to miss this signal.

- connected: if there is a gap in the series (a null value or values), then the line will skip the gap and connect to the next non-null value.
- null: if there is a gap in the series (a null value), then the line in the graph will be broken and show the gap.
- null as zero: if there is a gap in the series (a null value), then it will be displayed as a zero value in the graph panel.